### PR TITLE
Bugfix/cannot exit scheduler run

### PR DESCRIPTION
**Pull request by Yamsha75**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 apscheduler==3.6.3
+asyncio==3.4.3
```
Comment (dzkb) [inline requirements.txt:2]: Isn't `asyncio` standard Python library?

**Comment by dzkb** [comment]
Please update the README.md

File: requirements.txt (hunk 1 above)
Comment (Yamsha75) [reply requirements.txt]: True. Fixed

**Comment by Yamsha75** [comment]
> Please update the README.md

Done.

**Pull request merged**